package.json

```diff
@@ -327,8 +328,9 @@
   },
   "dependencies": {
     "@octokit/rest": "^15.9.5",
-    "es6-promise": "^4.2.5",
     "axios": "^0.18.0",
+    "debounce": "^1.2.0",
```

Comment:
Let's update `ThirdPartyNotices.txt` with the license text for this, and then I think this is ready to merge!